File: Back/ProAgil.Domain/Palestrante.cs
using System.Collections.Generic;

namespace ProAgil.Domain
{
    public class Palestrante
    {

        public int Id { get; set; }
        public string Nome { get; set; }
        public string Miniurriculo { get; set; }
        public string ImagemURL { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public List<RedeSocial> RedesSociais { get;}
        public List<PalestranteEvento> PalestrantesEventos { get;}
    }
}

File: Front/ProAgil-App/src/app/models/Palestrante.ts
import { Evento } from './Evento';
import { RedeSocial } from './RedeSocial';

export interface Palestrante {

  id : number;
  nome : string;
  miniurriculo : string;
  imagemURL : string;
  telefone : string;
  email : string;
  redesSociais : RedeSocial[];
  palestrantesEventos : Evento[];

}
